### DISCUSSION TITLE
0xcoinflasher/coin-flasher is a scam

### DISCUSSION THREAD

**Discussion by allreadydead11** [community/community#101906] [Other Feature Feedback, Questions, & Ideas]
Just thought you should be aware that 0xcoinflasher/coin-flasher is a scam. The main link to it is here https://github.com/0xcoinflasher/coin-flasher which being on git lends it some credibility and is not good for the git brand or its reputation and should be taken down. 
He is accepting btc for the software then just ghosting people as soon as he gets the conformation.

**Comment by queenofcorgis** [comment]
Hi @allreadydead11 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
